.github/workflows/publish-release-artifacts.yml

```diff
@@ -0,0 +1,180 @@
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+
+      - name: Set version env variable
+        id: set-version
+        run: |
+          RELEASE_VERSION=${{ github.event.inputs.ballerina_version }}
+          SHORT_VERSION=${{ github.event.inputs.short_version }}
+          CODE_NAME=${{ github.event.inputs.code_name }}
+          DISPLAY_VERSION="${{ github.event.inputs.display_version }}"
+
+          TAGGED_VERSION=$RELEASE_VERSION
+          LONG_VERSION=$RELEASE_VERSION-$CODE_NAME
+          if [ -n "${{ github.event.inputs.rc_suffix }}" ]; then
+            TAGGED_VERSION=$RELEASE_VERSION-${{ github.event.inputs.rc_suffix }}
+          fi
+
+          echo VERSION=$RELEASE_VERSION >> $GITHUB_ENV
+          echo SHORT_VERSION=$SHORT_VERSION >> $GITHUB_ENV
+          echo GIT_TAG=$TAGGED_VERSION >> $GITHUB_ENV
+          echo LONG_VERSION=$LONG_VERSION >> $GITHUB_ENV
+          echo DISPLAY_VERSION=$DISPLAY_VERSION >> $GITHUB_ENV
+          echo "::set-output name=version::$RELEASE_VERSION"
+          echo "::set-output name=taggedVersion::$TAGGED_VERSION"
+
+      - name: Download artifacts
+        run: |
+          baseUrl="https://github.com/ballerina-platform/ballerina-distribution/releases/download/v$GIT_TAG"
+          rm -rf $VERSION
+          mkdir $VERSION
+          cd $VERSION
+
+          echo "$baseUrl/ballerina-$LONG_VERSION.zip"
+
+          wget "$baseUrl/ballerina-$LONG_VERSION.zip"
+          wget "$baseUrl/ballerina-$SHORT_VERSION.zip"
+          wget "$baseUrl/ballerina-$LONG_VERSION-windows-x64.msi"
+          wget "$baseUrl/ballerina-$LONG_VERSION-linux-x64.deb"
+          wget "$baseUrl/ballerina-$LONG_VERSION-linux-x64.rpm"
+          wget "$baseUrl/ballerina-$LONG_VERSION-macos-x64.pkg"
+        env:
+          GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+          packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+          packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+          devCentralToken: ${{ secrets.BALLERINA_DEV_CENTRAL_ACCESS_TOKEN }}
+
+      - name: Generate metadata json
+        working-directory: ${{ steps.set-version.outputs.version }}
+        run: |
+          date=`date +%Y-%m-%d`
+          winInstSize=`ls -lh ballerina-$LONG_VERSION-windows-x64.msi | cut -d " " -f5 | sed 's/M/mb/g'`
+          linuxInstSize=`ls -lh ballerina-$LONG_VERSION-linux-x64.deb | cut -d " " -f5 | sed 's/M/mb/g'`
+          rpmInstSize=`ls -lh ballerina-$LONG_VERSION-linux-x64.rpm | cut -d " " -f5 | sed 's/M/mb/g'`
+          macInstSize=`ls -lh ballerina-$LONG_VERSION-macos-x64.pkg | cut -d " " -f5 | sed 's/M/mb/g'`
+
+          fileName=metadata.json
+          echo "{" >> $fileName
+          echo "	\"version\":\"$VERSION\"," >> $fileName
+          echo "	\"short-version\":\"$SHORT_VERSION\"," >> $fileName
```

Comment:
@NipunaMadhushan 
As now there us no short version from swan lake side shall we use same value